packages/opentelemetry-types/src/trace/span_context.ts

```diff
@@ -39,10 +38,13 @@ export interface SpanContext {
    * Trace options to propagate.
    *
    * It is represented as 1 byte (bitmap). Bit to represent whether trace is
-   * sampled or not.
+   * sampled or not. When set, the least significant bit documents that the
+   * caller may have recorded trace data. A caller who does not record trace
+   * data out-of-band leaves this flag unset.
+   *
    * SAMPLED_VALUE = 0x1 and NOT_SAMPLED_VALUE = 0x0;
    */
-  traceOptions?: TraceOptions;
+  traceOptions?: number;
```

Comment:
This should be limited to the set of possible values. The possible values should also be exported so that they can be used (for example `SpanContext.SAMPLED_VALUE`).

Reply:
Good idea! What about something like this:
```
type TraceOptionsUnsampled = 0x0;
type TraceOptionsSampled = 0x1;
type TraceOptions = TraceOptionsSampled|TraceOptionsUnsampled;
```

See https://basarat.gitbooks.io/typescript/docs/types/literal-types.html

Reply:
Done, PTAL